clang/test/CodeGenSYCL/address-space-conversions.cpp

```diff
@@ -0,0 +1,129 @@
+// RUN: %clang_cc1 -triple spir64 -fsycl-is-device -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
```

Comment:
Please add a comment here about what this test tests.

Reply:
Done.